openstudio-gems.gemspec

```diff
@@ -18,14 +18,14 @@ Gem::Specification.new do |spec|
   # gem version is specified in gemspec, gem source/location (e.g. github branch) can be specified in Gemfile
   # runtime dependency versions can be loosened while in development on branches if needed
   # runtime dependency versions should be specified as exact versions when merged to master or develop
-  spec.add_dependency 'openstudio-extension', '0.7.0'
-  spec.add_dependency 'openstudio-workflow', '2.3.1'
-  spec.add_dependency 'openstudio_measure_tester', '~> 0.3.2'
+  spec.add_dependency 'openstudio-extension', '0.8.0'
+  spec.add_dependency 'openstudio-workflow', '2.4.0'
+  spec.add_dependency 'openstudio_measure_tester', '~> 0.4.0'
 
   spec.add_dependency 'parallel', '1.19.1'
 
   # 20200324 TJC json native is enabled for all platforms and is included when building ruby
-  #spec.add_dependency 'json_pure', '2.2'
+  spec.add_dependency 'json_pure', '2.2'
```

Comment:
@jmarrec Was this intentional? I think everyone would prefer to use the native json, not the json_pure gem. It was [previously disabled](https://github.com/NREL/openstudio-gems/commit/a4da055ad9c1361634b96f9c3754fa3f28229e42).

Reply:
I probably messed up. @kbenne if you can revert if needed that'd be great